data/json/items/ammo/cannon.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "id": "cannon_shot",
+    "type": "AMMO",
+    "name": { "str": "cannon round shot" },
+    "description": "A bundle of wadding, black gunpowder, and cannonball, ready to be loaded into a cannon",
+    "weight": "4800 g",
+    "volume": "2500 ml",
+    "price": 11000,
+    "price_postapoc": 12000,
+    "material": [ "lead", "powder" ],
+    "symbol": "=",
+    "color": "white",
+    "count": 1,
+    "stack_size": 1,
+    "ammo_type": "cannon",
+    "range": 50,
+    "damage": { "damage_type": "bullet", "amount": 352 },
+    "//": "4 pound cannonball weighs 28000 grains,shot at 1215 ft/s by a 3in ordnance rifle, equals 124459 joules of muzzle energy according to Shooters Calculator, round down from square root 352.78",
+    "dispersion": 620,
```

Comment:
I always forget how dispersion is calculated, but a rifled muzzle loader is still going to have the worst dispersion possible.

Reply:
Yeah, I wasn't sure what was a good number for that so I copied it from the flintlock pistol. I'd also wanted to make it take longer to aim, but was unsure how to do that.